guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -15,24 +15,31 @@ The {ProjectWebUI} displays the status of the export.
 The status is not persistent; if you leave the status page, you cannot return to it.
 
 .CLI procedure
-. Clone a local copy of your Git repository:
+. Clone a local copy of your Git repository to a subdirectory of `/var/lib/foreman`:
```

Comment:
Why do we clone something? The templates plugin can also sync by itself and push.

https://github.com/theforeman/foreman_templates/blob/01d2dca4dd1deb7982cc6cd3a8f71e100f1b5282/app/services/foreman_templates/template_exporter.rb#L30-L61

So if you just use a git URL in the settings it should clone it, make the changes and push it back. That would remove all the need for directory permissions etc.

Reply:
@ekohl what would be exact steps here? Where is the setting to which I need to put the URL and what do I need to do after that to achieve the same result as it is at the end of this procedure?